package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "5.4.8",
+  "version": "6.0.0",
   "engines": {
-    "node": "^8.0.0 || ^10.13.0 || ^12.0.0"
+    "node": "^16.0.0"
```

Comment:
I personally saw no reason to support the earlier Node versions and, so, removed them. Let me know if you disagree. I think this may still work with Node 12 but I saw no reason to support earlier Node version.

Reply:
Node 14 still has a year left of maintenance support. Ideally we'd support both to allow for more incremental upgrades. I wouldn't invest more than an hour trying to troubleshoot issues that come up though

Reply:
Done. I updated to allow `^14.19` (latest Node 14 version) and `^16.14` (latest Node 16 version)